src/components/Home/Hero/ExchangeWidget/CoinInput/CoinInput.js

```diff
@@ -12,34 +12,43 @@ class CoinInput extends PureComponent {
   state = {
     value: '...',
     fetching: this.props.price.fetching,
+    focused: false
   };
 
   handleFocus = event => {
+    let value = event.target.value
     if (event.target.value === '...') {
-      this.setState({ value: '' });
```

Comment:
Why is this diff here @pragai ?

Reply:
Added new state: focused to handleFocus and handleBlur, so in componentDidUpdate there is no state re-set, and this focused state has to be set, too:

    if (event.target.value === '...') {
      value = '';
    }
    this.setState({ value: value, focused: true });